inspirehep/modules/workflows/utils/__init__.py

```diff
@@ -272,7 +272,7 @@ def get_document_url_for_reference_extraction(obj):
 
     url = documents[0]['url']
     obj.log.info('Using document with url "%s"', url)
-    return url
+    return current_app.config['INSPIREHEP_URL'] + url
```

Comment:
doesn't need a `/`?

Reply:
I don't think so, urls are starting with `/`

Reply:
[like `"url": "/api/files/be4ca558-1948-478c-a7f1-05af95b94282/2205.02190.pdf"`](https://github.com/inspirehep/inspire-next/blob/c42106fa7d47d89023748f62d0e900a839cc6976/inspirehep/modules/workflows/tasks/actions.py#L440)

Reply:
```
Instance: /usr/var/inspirehep-instance
In [1]: app.config['INSPIREHEP_URL']
Out[1]: 'http://10.254.122.111:80/api'
```

Reply:
https://github.com/cern-sis/kubernetes/blob/master/next/base/config/inspirehep.cfg#L32

Reply:
Maybe we should use something like https://github.com/inspirehep/inspire-next/blob/master/inspirehep/utils/url.py#L106-L114